kalastatic.module

```diff
@@ -234,3 +234,117 @@ function kalastatic_get_namespaces() {
   $settings = kalastatic_get_settings();
   return $settings['pluginOpts']['metalsmith-jstransformer']['engineOptions']['twig']['namespaces'];
 }
+
+/**
+ * Implements hook_page_attachments_alter().
+ */
+function kalastatic_page_attachments_alter(array &$page) {
+  // Unset Drupals default favicon.
+  $links = $page['#attached']['html_head_link'];
+  foreach ($links as $key => $link) {
+    if ($link[0]['href'] == '/core/misc/favicon.ico') {
+      unset($page['#attached']['html_head_link'][$key]);
+    }
+  }
+
+  // Handle custom favicons.
+  _kalastatic_favicon_head_tags($page);
```

Comment:
Rather than calling a custom function here, you could change `_kalastatic_favicon_head_tags()` to just implement `hook_page_attachments()`. The alter hook is more for removing or changing existing items rather than adding new, methinks.